templates/taxi/car_detail.html

```diff
@@ -9,4 +9,7 @@ <h1>Drivers</h1>
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+  <hr>
+    <a href="{% url 'taxi:car-update' pk=car.id %}">Update</a>
+    <a href="{% url 'taxi:car-delete' pk=car.id %}">Delete</a>
```

Comment:
Consider adding a 'Cancel' button here to lead the user back to the previous page, as per the checklist requirement: 'Add `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.'